openmetadata-service/src/main/java/org/openmetadata/service/secrets/converter/ClassConverterFactory.java

```diff
@@ -59,7 +58,6 @@ private ClassConverterFactory() {
             Map.entry(GcsConnection.class, new GcpConnectionClassConverter()),
             Map.entry(ElasticSearchConnection.class, new ElasticSearchConnectionClassConverter()),
             Map.entry(LookerConnection.class, new LookerConnectionClassConverter()),
-            Map.entry(OpenMetadataConnection.class, new OpenMetadataConnectionClassConverter()),
```

Comment:
since the `securityConfig` is not a `oneOf` anymore, we don't need an explicit converter